lib/patchelf/patcher.rb

```diff
@@ -173,13 +173,20 @@ def use_rpath!
     # Save the patched ELF as +out_file+.
     # @param [String?] out_file
     #   If +out_file+ is +nil+, the original input file will be modified.
+    # @param [Boolean] patchelf_compatible
+    #   When +patchelf_compatible+ is true, tries to produces the ELF same as the one produced by patchelf.
```

Comment:
"by Unix patchelf", to be accurate.

Reply:
Perhaps `NixOS/patchelf` to specify the C++ implementation? https://github.com/NixOS/patchelf

Reply:
@sjackman done the same, the file referred by github is old.